airflow/models.py

```diff
@@ -1494,6 +1494,9 @@ def get_template_context(self, session=None):
         prev_execution_date = task.dag.previous_schedule(self.execution_date)
         next_execution_date = task.dag.following_schedule(self.execution_date)
 
+        next_ds = next_execution_date.isoformat()[:10]
```

Comment:
How about we use `strftime` to make the formatting a little more explicit?

```
>>> from datetime import datetime
>>> dt = datetime.now()
>>> dt.isoformat()[:10]
'2018-05-22'
>>> dt.strftime('%Y-%m-%d')
'2018-05-22'
```

Reply:
Fixed.